docs/modules/getting-started/pages/get-started-binary.adoc

```diff
@@ -11,11 +11,11 @@ To complete this tutorial, you need the following:
 |===
 |Prerequisites|Useful resources
 
-|JDK 11+
+|JDK 17+
```

Comment:
To save upgrading this page in future - could we **just** put the supported JDK link instead? And skip versions here alltogether?